client/src/components/UploadModal.jsx

```diff
@@ -16,6 +16,14 @@ const PREVIEW_NUM = 6;
 const modalInactiveClass = "pt-4 px-2 text-xl inline txt-silver";
 const modalActiveClass = "pt-4 px-2 text-xl inline";
 
+const datasetTypes = [
```

Comment:
Can we add a few more for each kind of incident (Massage, Prostitution, Arrests)